### PR TITLE
feat: api response codes and +

### PR DESCRIPTION
**Pull request by carolinamenezes**
#### Types of changes
- [x] New content (guides, endpoints, app documentation)
- [ ] Improvement (make a documentation even better)
- [ ] Fix (fix a documentation error)
- [ ] Spelling and grammar accuracy (self-explanatory)

### CODE REVIEW

File: docs/guides/Getting-Started/best-practices-for-avoiding-rate-limit-errors.md [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+
+During this period, requests are categorized and handled with a deliberate focus on user needs and performance. For example, checkout requests are granted the highest priority. The store ensures that these requests are processed without any undue delays, allowing customers to successfully complete their purchases.
+
+In contrast, non-essential features, such as social sharing options, are granted the lowest priority, meaning these features may be temporarily deprioritized or even disabled. By applying this prioritization approach, the store assures that critical functions are available to users without interruption, even during peak traffic.
+
+## Implement optimized retries with exponential backoff
+
+Proactively manage your requests by avoiding unnecessary or excessive API calls. Excessive API calls can degrade your app's performance, increasing response times. If the service responds with rate limit exceeded errors, consider implementing exponential backoff, which gradually increases the delay between retry attempts to reduce the load on the service.
```
Comment (carolinamenezes) [inline docs/guides/Getting-Started/best-practices-for-avoiding-rate-limit-errors.md:80]:
```suggestion
Proactively manage your requests by avoiding unnecessary or excessive API calls. Excessive API calls can degrade your app's performance, increasing response times. If the service responds with rate limit exceeded errors, consider implementing exponential backoff, which gradually increases the delay between retry attempts to reduce the load on the service.

```

File: docs/guides/Getting-Started/best-practices-for-avoiding-rate-limit-errors.md [hunk 2]
```diff
@@ -0,0 +1,80 @@
...
+    - "/docs/guides/handling-errors-and-exceptions"
+    - "/docs/guides/vtex-io-documentation-engineering-guidelines"
+---
+
+Rate limits serve as crucial control mechanisms for managing the volume and frequency of requests made to an API. Exceeding these limits can lead to disruptions in service and a temporary suspension of access to [VTEX Core Commerce APIs](https://developers.vtex.com/docs/api-reference). This article delves into the best practices for responsibly dealing with rate limits while ensuring your apps and store code adhere to ethical and responsible digital practices.
+
+<!--->
+## Review API documentation
+
+The rate limits for the [VTEX Core Commerce APIs](https://developers.vtex.com/docs/api-reference) are detailed in the Rate limits article. Pay close attention to the rate limit policies, including the monthly quota and limits per second of the API you intend to use. Understanding these limits is the first step in avoiding rate limit errors.
```
Comment (ygorops) [inline docs/guides/Getting-Started/best-practices-for-avoiding-rate-limit-errors.md:20]: @carolinamenezes , we don't have this article.

**Review by ygorops — CHANGES_REQUESTED** [review]
A little adjustment required.

File: docs/guides/Getting-Started/best-practices-for-avoiding-rate-limit-errors.md [hunk 3]
```diff
@@ -0,0 +1,80 @@
...
+<!--->
+
```
Comment (carolinamenezes) [inline docs/guides/Getting-Started/best-practices-for-avoiding-rate-limit-errors.md:22]:
```suggestion
```

**Pull request merged**